optimade/validator/validator.py

```diff
@@ -662,14 +639,10 @@ def _construct_single_property_filters(
             if not test_value or (
                 isinstance(test_value[0], dict) or isinstance(test_value[0], list)
             ):
-                msg = f"Not testing queries on field {prop} of type {prop_type} with nested dictionary/list entries."
```

Comment:
So we're losing this message as well as others now? This should not happen, I think 😕

Reply:
We can keep this one (and the one below) if you want, but if this becomes a warning then I definitely don't think we return non-zero exit codes on warnings...

Reply:
Okay. Well let's discuss if we should do that or not? :) But perhaps for now we want it to be very boolean for sure. Is there no way of passing this information along without disrupting the desired outcome you're looking for here?